test/index.js

```diff
@@ -0,0 +1,25 @@
+import test from 'ava';
+import Sieve from '../sieve';
+import archive from './fixtures/archive';
+import folder from './fixtures/folder';
+
+test('archive to tree', t => {
+    const generatedTree = Sieve.toTree(archive.simple);
+    t.deepEqual(generatedTree, archive.tree);
+});
+
+// FIXME
```

Comment:
Is this failing because of
https://github.com/ProtonMail/sieve.js/issues/3 ?
If so, could you add that in the comment?

Reply:
Now all the tests are failing, the field `Vacation` is not optional anymore ? Also there are nulls generated in the tree "List".